healpy/src/_sphtools.pyx

```diff
@@ -83,7 +85,20 @@ def map2alm_spin_healpy(maps, spin, lmax = None, mmax = None):
     -------
     alms : list of 2 arrays
       list of 2 alms
+
+    Warnings
+    --------
+    This function previously made an exception for ``spin=0`` transforms, where
```

Comment:
Maybe just be blunt and say "This function previously returned undefined results when called with `spin=0`? Same for the alm2map direction.